operator/src/main/java/oracle/kubernetes/operator/helpers/CRDHelper.java

```diff
@@ -208,8 +231,54 @@ private boolean isOutdatedCRD(V1beta1CustomResourceDefinition existingCRD) {
       return COMPARATOR.isOutdatedCRD(existingCRD, this.model);
     }
 
+    private boolean isExistingCRDMissingVersion(V1beta1CustomResourceDefinition existingCRD) {
+      List<V1beta1CustomResourceDefinitionVersion> versions = existingCRD.getSpec().getVersions();
+      boolean found = false;
+      if (versions != null) {
+        for (V1beta1CustomResourceDefinitionVersion v : versions) {
+          if (KubernetesConstants.DOMAIN_VERSION.equals(v.getName())) {
+            found = true;
+            break;
+          }
+        }
+      }
+
+      return !found;
```

Comment:
i find the naming and negation confusing and suspect it will create future problems, would prefer to fix now